### ISSUE TITLE
Delete and/or Remove Previous Versions Needed

### ISSUE THREAD

**Issue by robertmcatee** [heraldry/armory#6]
I found no way to delete an artifact such as an Arms you no longer want, nor did I find a way to remove old/bad versions of artifacts.

**Comment by or** [comment]
Yes, there's no mechanism for that yet. Previous versions of charges, arms, or ribbons might be used in other entities, especially if they are public. Private entities can't be accessed by other users anyway, but I understand that they can pile up and get in the way.

Adding a proper user-driven deletion process for entities and accounts is on the TODO list, but in the meantime I can delete things manually if you message me on Discord.
